content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -229,12 +229,12 @@ can happen, according to:
 
 ## Pod disruption conditions {#pod-disruption-conditions}
 
-{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.26" state="beta" >}}
 
 {{< note >}}
-In order to use this behavior, you must enable the `PodDisruptionConditions`
+In order to use this behavior, you must have the `PodDisruptionConditions`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-in your cluster.
+enabled in your cluster.
```

Comment:
IMO, the previous version is more clear, but your change is also ok.

Reply:
ok, let's keep as is then.